applications/GeoMechanicsApplication/custom_python/geo_mechanics_python_application.cpp

```diff
@@ -62,6 +62,12 @@ PYBIND11_MODULE(KratosGeoMechanicsApplication,m)
 
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, DT_TEMPERATURE)
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, NORMAL_HEAT_FLUX)
+	KRATOS_REGISTER_IN_PYTHON_VARIABLE( m, AIR_TEMPERATURE )
```

Comment:
Is it indentation?

Reply:
I think this is a github issue, in CLion it looks okay:
![image](https://github.com/KratosMultiphysics/Kratos/assets/56549273/e5dbc3cc-8515-4a5f-9e86-efd491658cb0)

Reply:
Replaced a tab with spaces